TransferList/TransferList.js

```diff
@@ -539,15 +551,17 @@ const TransferListBase = kind({
 			list: 'first',
 			onSelect: setSelected,
 			selectedItems: selectedItems,
-			onSpotlightRight: handleSpotlightRight
+			onSpotlightRight: handleSpotlightRight,
+			showSelection: showSelectionOrder
```

Comment:
is there a reason why we cannot keep the same name "showSelectionOrder" ?

Reply:
I renamed it to 'showSelectionOrder' everywhere.